src/haconiwa/scan/comparator.py

```
@@ -0,0 +1,319 @@
+        for category, files in search_results['matches'].items():
+            for file_info in files:
+                model_info['files'].append(file_info)
+                model_info['size'] += file_info.get('size', 0)
+                
+                # Try to load config
+                file_path = self.base_path / file_info['path']
+                if file_path.name in ['config.json', 'model_config.json']:
+                    try:
+                        with open(file_path, 'r') as f:
+                            model_info['config'] = json.load(f)
+                    except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)